src/logic/addons.h

```diff
@@ -94,41 +98,46 @@ struct AddOnInfo {
 	std::function<std::string()> description;
 	std::function<std::string()> author;
 
-	AddOnVersion version;   // Add-on version (e.g. 1.2.3)
-	uint32_t i18n_version;  // (see doc/sphinx/source/add-ons.rst)
```

Comment:
Hmm I still dont get what this i18n_version is, from doc/sphinx/source/add-ons.rst ?

Reply:
The last sentence in the file:
> Each add-on has a translations version number in addition to the add-on version number; this allows the game to determine whether the translations for an installed add-on can be upgraded.

Basically, this is a version number which is stored on the server and increased automatically every time the translations for an add-on are updated. The number in the `AddOnInfo` states the version of the locally installed translation files for the add-on; if the server provides a higher number, then an upgrade of the translations is possible.